pyproject.toml

```diff
@@ -21,8 +21,8 @@ enum-tools = { extras = ["sphinx"], version = "0.9.0.post1", optional = true }
 crypto-cpp-py = "^1.0.4"
 marshmallow = "^3.15.0"
 marshmallow-oneofschema = "^3.0.1"
-cairo-lang = "0.10.2"
 typing-extensions = "^4.3.0"
+cairo-lang = "^0.10.3"
```

Comment:
Can we restrict it to a specific version like it was before?

Reply:
@Solpatium I remember we had a conversation about it once. Should we change it to `0.10.3`?

Reply:
Will change it to: https://python-poetry.org/docs/dependency-specification/#exact-requirements